app/templates/admin/userManagement.html

```diff
@@ -99,25 +99,18 @@ <h3 class="accordion-header" id="headingThree">
               <th>Reply-To-Email</th>
               <th>Sender Name</th>
             </tr>
-            <form action="" id="programSettings">
-                    <div>
-            <tr>
-                <td>
-                  <select class="form-select programData" id="programSelect" form="programSettings">
-                    <option selected disabled>--Choose a Program--</option>
-                    {% for program in programs %}
-                    <option value ="{{program.id}}" data-replytoemail="{{program.emailReplyTo}}" data-sendername="{{program.emailSenderName}}">{{program.programName}}</option>
-                    {% endfor %}
-                  </select>
-                </td>
-                <td><input type="text" name="Email" id="emailReplyTo" value=""></td>
-                <td><input type="text" name="Sender" value="" id="emailSenderName"></td>
-                <td>
-                  <button type="button" class="btn btn-success w-101" id="addNewProgramInfo" role ="button">Update</button>
-                </td>
+        <div class="programData" id="programSelect" form="programSettings">
```

Comment:
Divs around the table elements but not the whole table aren't a great idea. Do you need this div for anything? Could you put the id and class on other elements?